src/services/dialog-observer/simple-dialog-observer.ts

```
@@ -0,0 +1,298 @@
+            console.error(
+                `[SimpleDialogObserver] Error checking dialogs: ${error}`,
+            )
+        }
+    }
+
+    /**
+     * Simplified modal detection focusing on Google Meet's common patterns
+     */
+    protected async checkAndDismissModals(
+        page: Page,
+        customTimeout: number = 0,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)